src/AddTodoForm.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const AddTodoForm = () => {
+    return (
+        <div>
+            <form>
+                <label htmlFor="todoTitle">Title:</label>
+                <br />
+                <input type="text" id="todoTitle" required />
```

Comment:
Great job adding the `required` attribute! You can also add the `placeholder` attribute to make the intended use of the input even more explicit, ex:

`<input type="text" id="todoTitle" required placeholder='Enter title here' />`